test/signalrclient-e2e-tests/console_log_writer.cpp

```diff
@@ -0,0 +1,10 @@
+/// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "console_log_writer.h"
+
+void console_log_writer::write(const utility::string_t &entry)
+{
+	std::wcout << entry << std::endl;
```

Comment:
I don't think we should be writing to the console. This will pollute the output and CI won't be able to make any sense out of it. Logging to a file might be fine but I am not sure if it would be helpful (I noticed that people did not use log files written by functional tests in the managed client project)